controller/utils.go

```diff
@@ -126,6 +127,7 @@ func replacePodsAndWait(
 	wg.Add(len(podsChunk))
 	for _, pod := range podsChunk {
 		go func(pod v1.Pod) {
+			defer GinkgoRecover()
```

Comment:
It doesn't seem right to defer test related stuff in here. Why is this necessary?

Reply:
I need this to debug tests. It shouldn't have slipped through. My bad =(

Reply:
Removed